websockets/websockets.go

```diff
@@ -425,9 +424,6 @@ func (w *webSocket) loop() {
 	}
 }
 
-const binarytypeWarning = `You have not set a Websocket binaryType to "arraybuffer", but you got a binary response. ` +
```

Comment:
As the warning was already there, and we're moving towards making the extension non-experimental, and spec compliant, I guess it's fine to just add this "breaking change" (changing the default behavior), but I'm open for suggestions.